ios/FluentUI/Label/BadgeLabel.swift

```diff
@@ -29,6 +31,17 @@ class BadgeLabel: UILabel {
         isHidden = true
     }
 
+    override func didMoveToWindow() {
+        super.didMoveToWindow()
+
+        if shouldUseWindowColor {
```

Comment:
This could be a guard let to early return in case the property is false.

Reply:
Done!!